src/main/java/org/generationcp/middleware/domain/ontology/Variable.java

```diff
@@ -45,6 +45,8 @@
 	private Integer studies;
 	private Boolean hasPair;
   	private Boolean hasUsage;
+	private Integer germplasm;
```

Comment:
It seems to me we don't need to store the `germplasm `and `breedingMethods` usage counts within this object. 
They are just used as a factor to computing `hasUsage` property right?

Reply:
@darla-leafnode 
It's used to return in the Metadata.